models/kyberswap/kyberswap_aggregator_trades.sql

```diff
@@ -0,0 +1,58 @@
+{{ config
+(
+    tags=['dunesql'],
+    alias = alias('aggregator_trades'),
```

Comment:
Maybe kyberswap_aggregator.trades
```suggestion
    schema = 'kyberswap_aggregator',
    alias = alias('trades'),
```

Reply:
Fixed.